cmd/cloud/deployment_test.go

```diff
@@ -274,3 +274,151 @@ func TestDeploymentVariableUpdate(t *testing.T) {
 	assert.Contains(t, resp, "test-value-2-update")
 	mockClient.AssertExpectations(t)
 }
+
+func TestNewDeploymentWorkerQueueRootCmd(t *testing.T) {
+	expectedHelp := "Manage worker queues for an Astro Deployment."
+	testUtil.InitTestConfig(testUtil.CloudPlatform)
+	buf := new(bytes.Buffer)
+
+	t.Run("worker-queue command runs", func(t *testing.T) {
+		wQueueCmd := newDeploymentWorkerQueueRootCmd(os.Stdout)
+		wQueueCmd.SetOut(buf)
+		_, err := wQueueCmd.ExecuteC()
+		assert.NoError(t, err)
+		assert.Contains(t, buf.String(), "worker-queue")
+	})
+
+	t.Run("-h prints worker-queue help", func(t *testing.T) {
+		cmdArgs := []string{"worker-queue", "-h"}
+		resp, err := execDeploymentCmd(cmdArgs...)
+		assert.NoError(t, err)
+		assert.Contains(t, resp, expectedHelp)
+	})
+}
+
+func TestNewDeploymentWorkerQueueCreateCmd(t *testing.T) {
+	expectedHelp := "Create a worker queue for an Astro Deployment"
+	testUtil.InitTestConfig(testUtil.CloudPlatform)
+	mockClient := new(astro_mocks.Client)
+	astroClient = mockClient
+	deploymentRespNoQueues := []astro.Deployment{
+		{
+			ID:             "test-deployment-id",
+			Label:          "test-deployment-label",
+			RuntimeRelease: astro.RuntimeRelease{Version: "4.2.5"},
+			DeploymentSpec: astro.DeploymentSpec{
+				Workers: astro.Workers{
+					AU: 10,
+				},
+				Scheduler: astro.Scheduler{
+					AU:       5,
+					Replicas: 3,
+				},
+			},
+			WorkerQueues: []astro.WorkerQueue{},
+		},
+		{
+			ID:             "test-deployment-id-1",
+			Label:          "test-deployment-label-1",
+			RuntimeRelease: astro.RuntimeRelease{Version: "4.2.5"},
+			DeploymentSpec: astro.DeploymentSpec{
+				Workers: astro.Workers{
```

Comment:
PS we can get rid of all the worker AU invocations, the field is deprecated

Reply:
@frank-ye we will take care of it shortly in this [issue](https://app.zenhub.com/workspaces/managed-airflow---data-practitioner-6258e7ff010e82001090f49a/issues/astronomer/astro-cli/639)